bears/xml/XMLBear.py

```diff
@@ -22,12 +22,13 @@ class XMLBear(LocalBear, Lint):
     diff_message = "XML can be formatted better."
     output_regex = r'(.*\.xml):(?P<line>\d+): (?P<message>.*)\n.*\n.*'
     gives_corrected = True
+    use_stderr = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
             return self._process_issues(self.stderr_output, filename)
+        if self.stdout_output:
+            return self._process_corrected(self.stdout_output, filename, file)
```

Comment:
If this is a bug that you're trying to fix, please make a new test to test this.

Either way it would be good to add a comment here stating this so that someone else doesnt change it later by mistake
Note: You can use `elif` instead of `if` here.